css/responsive/desktop.styl

```diff
@@ -94,4 +94,23 @@ $margin-large = ($gutter * 5)
 // BASIC
 
 #main-content
-  padding-top: 58px + $margin-tiny
\ No newline at end of file
+  padding-top: 58px + $margin-tiny
+
+// ******************************************************
+
+// HEADER
+
+#header-menu
+  padding-top: 0
+  padding-bottom: 0
+  font-size: $font-size-h3
+  display: initial
+  li
+    display: inline-block
```

Comment:
we know the amount of items in the list so you could use nth selectors rather than switching the :after/:before. But float is fine as long as `#header-menu` has a clearfix added